paddle/fluid/operators/mul_op.h

```diff
@@ -110,95 +112,5 @@ class MulGradKernel : public framework::OpKernel<T> {
   }
 };
 
-template <typename DeviceContext, typename T>
-class MulDoubleGradKernel : public framework::OpKernel<T> {
```

Comment:
Why was this operator completely removed?

Reply:
that's because int8 mul is only used for inference

Reply:
Yeah, but that's grad code for general case which includes fp32.